### PR TITLE
1658 desktop enterprise

### LINKED ISSUE #1658: /desktop/enterprise/ to VBT

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* update /desktop/enterprise to vbt

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/desktop/enterprise)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/5.%20Desktop?preview=desktop-enterprise.png) and the [demo](http://www.ubuntu.com-1658-desktop-enterprise.demo.haus/desktop/enterprise)

## Issue / Card

Fixes #1658

### CODE REVIEW

**Comment by pmahnke** [comment]
@nottrobin 

links are according to the design
image

the text is probably not right, but I have proposed an 'is-compact' pull-quote for Yaili to help direct?  the suggestion on the design was to use just a normal block quote, but it has a border-left which looks really strange.

**Comment by nottrobin** [comment]
@pmahnke okay I'm approving my changes and leaving it up to design review.

**Comment by pmahnke** [comment]
@nottrobin and @yaili here it is with h3's sized h4 and the blockquote with a removed border-left... still looks a bit funny, but the headings are better

image

**Comment by yaili** [comment]
Design review:
- [ ] There should be a space between the link text and the external link icon in "Ubuntu in action"
- [ ] The text under the title "Bukwang Pharmaceuticals" is a paragraph, right? It should be just a p
- [ ] I think we should use existing heading size utility classes to reduce the size of pull quote text for now (heading 5)

**Comment by pmahnke** [comment]
@yaili updated... I am still using the is-compact, I just made the paragraph include .p-heading--five

I also dropped the size of the h3s to p-heading--four for the Ubuntu in action section

image

**Comment by yaili** [comment]
@pmahnke is the sass file referring to the p-heading--five class or is it manually adding the font-size: 1.313rem? If it's the latter, I don't think we should create another instance of font-sizes that we'll have to update later.

**Comment by pmahnke** [comment]
@yaili the scss is pulling in the p-heading--five styles... it is also doing a few other things, like adjusting the size of the quotes. also, you should look on other pages that use this style... like /desktop/government (in that PR)  I can remove if you want and don't think we will get this in vbt

```scss
.p-pull-quote.is-compact,
  .p-pull-quote--accent.is-compact {
    > p {
      @include heading-5;

      &:first-of-type::before,
      &:last-of-type::after {
        font-size: $sp-large;

        @media (min-width: $breakpoint-medium) {
          font-size: $sp-xx-large;
        }
      }
    }

    .p-pull-quote__citation {
      font-size: 1.1rem;
      line-height: 1.15;
      margin-top: 2rem;
    }
  }
```

**Comment by yaili** [comment]
@pmahnke I just had a look at the pull quote in https://docs.vanillaframework.io/en/patterns/pull-quote and it doesn't seem it's been implemented according to [spec](https://github.com/ubuntudesign/vanilla-design/tree/master/Pull%20quote) - the font size doesn't change and it should - I guess we should only make changes to this once the pull quote is correct, right?

**Comment by pmahnke** [comment]
@yaili I think pull-quotes are responsive, meaning the screen size... NOT being used in a col-4 space... not sure what I should do at this point... I would prefer to go with mine and propose back into Vanilla?

File: templates/desktop/enterprise.html [hunk 1]
```diff
@@ -1,89 +1,93 @@
-{% extends "desktop/base_desktop.html" %}
+{% extends "desktop/base_desktop-v1.html" %}
 
 {% block title %}For enterprise{% endblock %}
 {% block extra_body_class %}desktop-enterprise{% endblock %}
```
Comment (barrymcgee) [inline templates/desktop/enterprise.html]: @pmahnke Is this class still needed?

Comment (pmahnke) [inline templates/desktop/enterprise.html]: @barrymcgee I don't know... but @nottrobin does

Comment (nottrobin) [inline templates/desktop/enterprise.html]: I don't actually know about any of the `extra_body_class`es. But personally I'd just grep the Sass and see if it's mentioned. If not, go ahead and remove it.

**Comment by yaili** [comment]
When I looked at Vanilla docs they didn't seem to be responsive in any case. I think we can go with yours, but it shouldn't be proposed back in Vanilla before there's a design issue completed to define correct specs.

**Pull request merged**